package.json

```diff
@@ -1,53 +1,53 @@
 {
-  "name": "smart-shopping-list",
-  "version": "0.1.0",
-  "private": true,
-  "dependencies": {
-    "firebase": "^7.6.1",
-    "husky": "^3.1.0",
-    "lint-staged": "^9.5.0",
-    "prettier": "^1.19.1",
-    "react": "^16.12.0",
-    "react-dom": "^16.12.0",
-    "react-firestore": "^1.0.1",
-    "react-router": "^5.1.2",
-    "react-router-dom": "^5.1.2",
-    "react-scripts": "3.3.0"
-  },
-  "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject",
-    "format": "prettier --write 'src/**/*.{js,jsx,ts,tsx,json,css,scss,md}'"
-  },
-  "eslintConfig": {
-    "extends": "react-app"
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged"
+    "name": "smart-shopping-list",
```

Comment:
Not a huge deal, but watch your formatter settings so you don't inadvertently add more files than intended or create noise in your PR. This change looks like it's just adding an extra indent.